courses/free-courses-en.md

```diff
@@ -316,10 +316,10 @@
 * [Git](https://www.youtube.com/playlist?list=PLFBirL3MAv29Vy_L7MmV2QaZLvAadFPHR) - Gwendolyn Faraday
 * [Git & GitHub Tutorial for Beginners](https://www.youtube.com/playlist?list=PL4cUxeGkcC9goXbgTDQ0n_4TBzOO0ocPR) (The Net Ninja)
 * [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV)
+* [GIT for beginners](https://geeksgod.com/udemy-free-course/git-for-beginners/) - Maksym Rudnyi (Eduonix Learning Solutions)
```

Comment:
Following guidelines:
- tracking sites / parameters are not accepted. The right link is https://www.eduonix.com/git-for-beginners
- maybe complete it with note about need platform sign up

So, try to fix it with:

```markdown
* [GIT for beginners](https://www.eduonix.com/git-for-beginners) - Maksym Rudnyi (Eduonix Learning Solutions) *(account or email address required)*
```